### PR TITLE
added index.html

### PR DESCRIPTION
**Pull request by omukas-worldremit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # FirstRepo
 
 Added important information
+
```
Comment (omukas-worldremit) [inline README.md]: why do you need that `one`??

**Comment by eriksleziona** [comment]
Hello

File: README.md (hunk 1 above)
Comment (Olacz82) [reply README.md]: :)

Comment (tytzhynsoo) [reply README.md]: All is ok, man

**Comment by KozaX** [comment]
:__:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <style>
+
+        .box-1 {
+            /*padding: 10px;*/
+            border: solid red;
+        }
+
+        div.box-2 {
+            margin: 10px;
+            border: solid blue;
```
Comment (YevheniiaHalkina) [inline index.html]: Nice color!

File: README.md (hunk 1 above)
Comment (smelnikava) [reply README.md]: Sure:)

Comment (omukas-worldremit) [reply README.md]: Image of Yaktocat

```java
public static void main() { }
```

Comment (eriksleziona) [reply README.md]: Image of Yaktocat

**Pull request merged**